### ISSUE TITLE
File is not recognized on windows

### ISSUE THREAD

**Issue by murawskikrzysiek** [anthraxx/intellij-awesome-console#16]
The file is also not detected on windows. 
When the path is:
C:\your\file\structure\file1 it will not be recognized
When the path is:
C:/your/file/structure/file1 it will be recognized

Note that all os methods return the first version (like getcwd, abspath etc)

**Comment by murawskikrzysiek** [comment]
Actually the problem is with the first slash only. 
![screen shot 2016-02-01 at 11 54 29](https://cloud.githubusercontent.com/assets/6305746/12715442/a36fa940-c8da-11e5-9043-6556dd14d3c2.png)

**Comment by anthraxx** [comment]
we believe this issue is fixed in the 0.1337.6 release, which should appear soon in the plugin repositories (plugin already uploaded).
If you still encounter issues, please notify us about it.

**Comment by mbjelac** [comment]
Not working for me in WebStorm (2017.2.5 Build #WS-172.4343.25), plugin version: 0.1337.6

Source URL example (extracted with 'copy URL', dummified sensitive data):

```
C:/paths/delimited/with/slashes/dash-delimited-name.component.ts:17:21)
```

Error message:

```
Windows cannot find
'file:///C:\paths\delimited\with\slashes\dash-delimited-na...:21).
Make sure you ... blah blah.
```

What's with the trailing closing bracket?
